### DISCUSSION TITLE
Implement place(x,y)

### DISCUSSION THREAD

**Discussion by fcastellanos1** [TomSchimansky/CustomTkinter#1749] [Ideas]
Hello!

You guys are awesome, and I really wanted to thank you for all the work you must have been through. On the other hand, I wanted to suggest that you implement the place(x,y) method, which is really useful for placing objects at a specific coordinates without having to use tricks with grid() or pack(). It would also be useful to be able to use pack() and grid() in the same master.

Thank you!

**Comment by ElectricCandlelight** [comment]
Place() already works, I'd advise not using it. 

As with normal tkinter you should pick one layout system per container.
